pkg/config/app/kuma-dp/config.go

```diff
@@ -113,6 +118,9 @@ func (s SDS) Validate() error {
 	default:
 		return errors.Errorf(".Type has to be either %s or %s", SdsCp, SdsDpVault)
 	}
+	if s.Address == "" && strings.HasPrefix(s.Address, "unix:///") {
```

Comment:
That condition doesn't look correct, at least because we check prefix if address is empty